src/OSPSuite.Core/Domain/SensitivityAnalyses/OutputParameterSensitivity.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace OSPSuite.Core.Domain.SensitivityAnalyses
+{
+   public class OutputParameterSensitivity
```

Comment:
So this represents one output values (float array) for a given value of a parameter
Parameter is identified by path 
QuantityValues is the calculated output
ParameterValue is the value of said parameter
QuantityPath is path of output.

I might rename QuantityValues => OutputValues, QuantityPath =>OutputPath